purchases/src/main/kotlin/com/revenuecat/purchases/paywalls/PaywallData.kt

```diff
@@ -0,0 +1,206 @@
+     */
+    val assetBaseURL: URL,
+    internal val localization: Map<String, LocalizedConfiguration>,
+) {
+    /**
+     * Generic configuration for any paywall.
+     */
+    data class Configuration(
+        /**
+         * The list of package identifiers this paywall will display.
+         */
+        val packages: List<String>,
+
+        /**
+         * The package to be selected by default.
+         */
+        val defaultPackage: String? = null,
+
+        /**
+         * The images for this template.
+         */
+        val images: Images,
+
+        /**
+         * Whether the background image will be blurred (in templates with one).
+         */
+        val blurredBackgroundImage: Boolean = false,
+
+        /**
+         * Whether a restore purchases button should be displayed.
+         */
+        val displayRestorePurchases: Boolean = true,
+
+        /**
+         * If set, the paywall will display a terms of service link.
+         */
+        val termsOfServiceURL: URL? = null,
+
+        /**
+         * If set, the paywall will display a privacy policy link.
+         */
+        val privacyURL: URL? = null,
+
+        /**
+         * The set of colors used.
+         */
+        val colors: ColorInformation,
+    ) {
+        data class Images(
+            /**
+             * Image displayed as a header in a template.
+             */
+            val header: String? = null,
+
+            /**
+             * Image displayed as a background in a template.
+             */
+            val background: String? = null,
+
+            /**
+             * Image displayed as an app icon in a template.
+             */
+            val icon: String? = null,
+        )
+
+        data class ColorInformation(
+            /**
+             * Set of colors for light mode.
+             */
+            val light: Colors,
+
+            /**
+             * Set of colors for dark mode.
+             */
+            val dark: Colors? = null,
+        )
+
+        data class Colors(
+            /**
+             * Color for the background of the paywall.
+             */
+            val background: PaywallColor,
+
+            /**
+             * Color for the primary text element.
+             */
+            val text1: PaywallColor,
+
+            /**
+             * Color for secondary text element.
+             */
+            val text2: PaywallColor? = null,
+
+            /**
+             * Background color of the main call to action button.
+             */
+            val callToActionBackground: PaywallColor,
+
+            /**
+             * Foreground color of the main call to action button.
+             */
+            val callToActionForeground: PaywallColor,
+
+            /**
+             * Primary accent color.
+             */
+            val accent1: PaywallColor? = null,
+
+            /**
+             * Secondary accent color.
+             */
+            val accent2: PaywallColor? = null,
```

Comment:
Same, these are  `= null` so there's a constructor without them. I saw the iOS code did this and thought it was convenient